maps/away/ships/pra/headmaster/headmaster_ship.dm

```diff
@@ -93,10 +109,16 @@
 	fore_dir = NORTH
 	vessel_size = SHIP_SIZE_TINY
 
-/obj/machinery/computer/shuttle_control/explore/headmaster_shuttle
+/obj/machinery/computer/shuttle_control/explore/terminal/headmaster_shuttle
 	name = "shuttle control console"
 	shuttle_tag = "Orbital Fleet Shuttle"
-
+	icon = 'icons/obj/machinery/modular_terminal.dmi'
+	icon_screen = "helm"
+	icon_keyboard = "security_key"
+	is_connected = TRUE
+	has_off_keyboards = TRUE
+	can_pass_under = FALSE
+	light_power_on = 1
```

Comment:
some of these vars are already defined on `/obj/machinery/computer/shuttle_control/explore/terminal`
remove the re-definitions